spec/savon/request_spec.rb

```diff
@@ -165,9 +166,6 @@ def new_wsdl_request
 
       describe "set with a valid decrypting password" do
         it "handles SSL private keys properly" do
-          if RUBY_ENGINE == 'jruby'
-            pending("find out why this fails with a null pointer exception on jruby")
```

Comment:
This way of setting a test to Pending led to a Deprecation warning being printed in the test.